example/README.md

```diff
@@ -85,3 +85,34 @@ your stack name [here](../travis-ci/select-stack.js).
 There are tests for redeploying the Cumulus stack while a workflow is running (see `spec/redeployment`). This is acheived by backing up with the `workflows.yml` file, updating it, and redeploying the stack. When redeploy tests are complete, the original `workflows.yml` is restored, the backup file is deleted, and the stack is redeployed, restoring it to its original state.
 
 Please note that the stack will be redeployed multiple times when running tests and any errors during redeployment can result in errors in later tests. The deployment output is printed to the console.
+
+## Cumulus Documentation
```

Comment:
This is in `example/README` and should go in the `README` in the top level cumulus directory.